### DISCUSSION TITLE
Can a button be added to independently control the dark mode of G-Helper interface?

### DISCUSSION THREAD

**Discussion by asolouis** [seerge/g-helper#3364] [Ideas]
Can a button be added to independently control the dark mode of G-Helper interface?

**Comment by seerge** [comment]
@asolouis hello, there is no button (as it's quite niche setting). But you can override default theme via [config as explained in Power User settings](https://github.com/seerge/g-helper/wiki/Power-user-settings#override-ui-theme) :)

**Reply by asolouis** [reply]
Got it, thank you for your reply. The dark mode has been successfully modified.
